saleor/channel/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+from ..core.permissions import ChannelPermission
+
+
+class Channel(models.Model):
+    name = models.CharField(max_length=250)
+    slug = models.SlugField(max_length=255, unique=True)
+    currency_code = models.CharField(max_length=128)
```

Comment:
Maybe `max_length` could be smaller like `20`? We have now `DEFAULT_CURRENCY_CODE_LENGTH=3` setting, cryptos are max 4 characters if I'm correct so 20 is still safe.